File: 0x01-variables_if_else_while/1-last_digit.c
#include <stdio.h>
#include <stdlib.h>
#include <time.h>
/* more headers goes there */


/* betty style doc for function main goes there */
/**
 * main - Entry point of the program
 *
 * This function serves as the entry point for the program execution.
 * It prints the string "Programming is fun!" using the puts function.
 * The program then returns 0 to indicate successful execution.
 *
 * Return: Always 0 (Success)
 */
int main(void)
{
	int n;
	int lastDigit;

	srand(time(0));
	n = rand() - RAND_MAX / 2;
	/* your code goes there */
	lastDigit = n % 10;
	
	printf("Last digit of %d is %d ", n, lastDigit);
	if (lastDigit>5)
	{
		printf("and is greater than 5\n");
	}
	else if (lastDigit==0)
	{
		printf("and is 0\n");
	}
	else if (lastDigit<6)
	{
		printf("and is less than 6 and not 0\n");
	}
	return (0);
}

File: 0x01-python-if_else_loops_functions/1-last_digit.py
#!/usr/bin/python3
import random
number = random.randint(-10000, 10000)
dig = int(str(number)[-1:])
print(f'Last digit of {number:d} is {dig:d}', end=' ')
if dig > 5:
    print('and is greater than 5')
elif dig < 6:
    if dig == 0:
        print('and is 0')
    else:
        print('and is less than 6 and not 0')
